VoxelSniperCore/src/main/java/com/github/kevindagame/command/VoxelCommand.java

```diff
@@ -24,7 +24,12 @@ public VoxelCommand(String name) {
 
     public boolean execute(IPlayer player, String[] args) {
         if (getPermission() == null || getPermission().isEmpty() || player.hasPermission(getPermission())) {
-            return doCommand(player, args);
+            try {
+                return doCommand(player, args);
+            } catch (Exception e) {
```

Comment:
I would prefer a less generic Exception type here, what kinds of Exceptions doe we expect?

Reply:
This could be anything. It's just a catch all to prevent it from being silently swallowed by the command executor (in case of a java.io.EOFException, it got swallowed, giving a message in chat, but no stacktrace

Reply:
But shouldn't we also let the player know something failed? I think it would now fail silently from a user perspective.

Reply:
https://github.com/KevinDaGame/VoxelSniper-Reimagined/pull/233/commits/1a8211ea4592fceed3d265bc198a907e01ec0939